github/examples_test.go

```diff
@@ -101,3 +101,40 @@ func ExamplePullRequestsService_Create() {
 
 	fmt.Printf("PR created: %s\n", pr.GetHTMLURL())
 }
+
+func ExampleOrganizationsService_ListTeams() {
+	// This example shows how to get a team ID corresponding to a given team name
+
+	// Note that authentication is needed here as you are performing a lookup on
+	// an organization's administrative configurationC, so you will need to modify
+	// the example to provide an oauth client to github.NewClient() instead of nil.
+	// See the following documentation for more // information on how to
+	// authenticate with the client:
+	// https://godoc.org/github.com/google/go-github/github#hdr-Authentication
+	client := github.NewClient(nil)
+
+	teamID := int64(-1)
+	teamName := "Developpers team"
+	ctx := context.Background()
+	opts := &github.ListOptions{}
+
+	for {
+		teams, _, err := client.Organizations.ListTeams(ctx, "myOrganization", opts)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+		if len(teams) == 0 {
```

Comment:
Hmmm... I'm not sure that this is an excellent example for how to paginate because it makes one extra call to the API when it could know up front how many pages are expected by parsing the results of the first call.

See the `Link:` information located here: https://developer.github.com/v3/teams/#list-teams
and here: https://github.com/google/go-github/blob/master/github/github.go#L358-L366